read-the-docs/source/Bugzilla.rst

```diff
@@ -24,6 +25,17 @@ Methods
 -  `add_cc() <#add_cc>`__
 -  `add_attachment() <#add_attachment>`__
 
+get_ticket_content()
+--------------------
+
+``get_ticket_content(self, ticket_id=None)``
+
+Queries the Bugzilla API to get ticket_content using ticket_id.
+
+.. code:: python
+
+    t = ticket.create(ticket_id='Ticket_ID')
```

Comment:
Use '<ticket_id>' instead of 'Ticket_ID'.

Reply:
changed